_docs_operate/configuration.md

```diff
@@ -88,7 +88,7 @@ The Connector provides the following configuration parameters:
 {
     "debug": false,
     "transportLibrary": {
-        "baseUrl": "https://prod.enmeshed.eu",
+        "baseUrl": "BASE_URL",
```

Comment:
use the same wording as in 60-addresses.md: backboneHostname?

Reply:
no - it's a different thing.

indeed the backbone hostname for the address generation will be extracted from the baseUrl

Reply:
baseUrl: https://prod.enmeshed.eu
hostname: prod.enmeshed.eu